mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+  let salesTaxForProduct = (productPrice + ((productPrice * 20) / 100))
+  return salesTaxForProduct;
+ };
```

Comment:
Comment: Well done. Here is another way to consider 
function calculateSalesTax(sales) {
  let tax = sales * 0.2;
  let total = sales + tax;
  return total;
}